src/main/java/net/pms/formats/ATRAC.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Universal Media Server, for streaming any medias to DLNA
+ * compatible renderers based on the http://www.ps3mediaserver.org.
+ * Copyright (C) 2012 UMS developers.
+ *
+ * This program is a free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; version 2
+ * of the License only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package net.pms.formats;
+
+public class ATRAC extends AudioFile {
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public Identifier getIdentifier() {
+		return Identifier.ATRAC;
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public String[] getSupportedExtensions() {
+		return new String[] {
+			"at3",
+			"oma",
+			"aa3",
+			"at9",
```

Comment:
@Nadahar That's what happen with Apple, Sony and all their proprietary staff...money and mess...

I think that it'll need to be tested first, is one of these work.
FFMpeg don't support the avanced one:
https://trac.ffmpeg.org/ticket/5334

Here some .at3 samples:
https://www.djxmmx.net/ffmpeg/5334/

And here a Sony rules to convert them to MP3:
http://www.sony.co.uk/support/en/content/cnt-tut/shared/convert-atrac-to-mp3